dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -300,7 +300,7 @@ private static VersionRangeResult findVersionRange(
       String artifactId)
       throws MavenRepositoryException {
 
-    Artifact artifactWithVersionRange = new DefaultArtifact(groupId, artifactId, null, "(0,]");
+    Artifact artifactWithVersionRange = new DefaultArtifact(groupId, artifactId, null, "(,]");
```

Comment:
As per https://cwiki.apache.org/confluence/display/MAVENOLD/Dependency+Mediation+and+Conflict+Resolution#DependencyMediationandConflictResolution-DependencyVersionRanges, both min and max are optional. We didn't have to specify the minimum limit.